.cloudflare/build.sh

```diff
@@ -1,4 +1,17 @@
-# !/bin/bash
+#!/bin/bash
 
-  npm run write-items
-  npm run build
+# Echo all env vars for debugging
+echo "DISCORD_CLIENT_ID: $DISCORD_CLIENT_ID"
+echo "DISCORD_CLIENT_SECRET: $DISCORD_CLIENT_SECRET"
+echo "NODE_VERSION: $NODE_VERSION"
+echo "SITE_NAME: $SITE_NAME"
+
+# Write wrangler.toml dynamically
+cat <<EOF > wrangler.toml
+[vars]
+DISCORD_CLIENT_ID = "$DISCORD_CLIENT_ID"
```

Comment:
This doesnt work, we have these in cloudflare secrets but the `Deploying to Cloudflare's global network` step cant access them.